### ISSUE TITLE
treemacs broken

### ISSUE THREAD

**Issue by shakor0** [Alexander-Miller/treemacs#1046]
After todays update my treemacs broke:
GNU Emacs 29.1
Error in post-command-hook (treemacs--post-command): (wrong-type-argument number-or-marker-p nil)

**Comment by sebastiaanspeck** [comment]
Will be fixed with https://github.com/Alexander-Miller/treemacs/pull/1044

**Comment by shakor0** [comment]
That fixes it indeed, thanks.

**Comment by apoorv569** [comment]
> Will be fixed with #1044

Hi, I just updated my `Emacs` packages today and it seems to be still broken for me. I am getting this error,
```
Error in post-command-hook (treemacs--post-command): (wrong-type-argument number-or-marker-p nil)
```

I even removed all my packages and re-installed but no luck.  I am using the `nativecomp` version as well and on `Emacs 29.1`.
